### ISSUE TITLE
flag potential duplicates in ForC_simplified

### ISSUE THREAD

**Issue by ValentineHerr** [forc-db/ForC#210]
## - create age group 0-10-50-100-999
groups will be:
-[0-10]
-]10-50]
-]50-100]
-]100-999]
-unknown  (NA, NAC, NI...)

## - split by variable, geographic_area, year and age group
considering unknown year as its own group, and unknown age group as its own group too

## flag with these rules
- if any GROA: flag all others
- if only GROA: keep all
- if no GROA: 
   - if any ForC and any SRDB:
         - flag all SRDB
         - apply rules for "if only ForC" below to remaing ForC records
   - **if only SRDB: keep all (or do the same as if only ForC below?)**
   - if only ForC: 
        - if variable is GPP or NEP or Reco, keep most recent citation (randomly pick one if ties)
        - if variable is not GPP nor NEP nor Reco, keep oldest citation (randomly pick one if ties)

@teixeirak, can you confirm what is in bold above?

**Comment by teixeirak** [comment]
>     * if only SRDB: keep all (or do the same as if only ForC below?)

no, please apply the same rules as for ForC only. I'll edit so the outline above.

**Comment by ValentineHerr** [comment]
grouping by year:
@teixeirak, how to handle "date" "start.date" and "end.date"  ? take "start date" as the "date"?

**Comment by teixeirak** [comment]
yes

**Comment by ValentineHerr** [comment]
@teixeirak, there are a few missing citation IDs (see below). Should I just take the  date of `loaded.from` for this task?

![image](https://user-images.githubusercontent.com/19673377/94573187-d8e4ca80-023f-11eb-80b8-d1e14236fadb.png)

**Comment by ValentineHerr** [comment]
actually, loaded.from doesn't exist in ForC_simplified.... can we fill in citation ID in the measurements table? that would make my life easier...

**Comment by teixeirak** [comment]
> @teixeirak, there are a few missing citation IDs (see below). Should I just take the date of `loaded.from` for this task?
> 
> ![image](https://user-images.githubusercontent.com/19673377/94573187-d8e4ca80-023f-11eb-80b8-d1e14236fadb.png)

sure.

**Comment by teixeirak** [comment]
> actually, loaded.from doesn't exist in ForC_simplified.... can we fill in citation ID in the measurements table? that would make my life easier...

Let's just put Taylor under citation ID. 

Please note that I just pushed some changes to the data (resolving root biomass outlier), so be sure to pull the most recent version.

**Comment by ValentineHerr** [comment]
ok, thanks!

**Comment by ValentineHerr** [comment]
I made the change in the citation ID so don't forget to pull if you are editing the file more.

**Comment by ValentineHerr** [comment]
I pushed a new ForC_simplified with new "suspected.duplicate" column. let me know what you think.

It looks like it will remove 5911 of the non-managed, non-disturbed, non-no.history.info records (but there is >10000 that are flagged).

I'll work on updating the figures.

**Comment by teixeirak** [comment]
Awesome! I think we've found a good solution.

**Comment by teixeirak** [comment]
@ValentineHerr , based on my current understanding of the process, we're potentially giving preference to GROA records with now info on veg.type, and thereby losing data altogether. For the next round, we may try to address that.

**Comment by teixeirak** [comment]
We could also refine this to compare the estimates-- true duplicates should be the same or similar.
